recipes/pulchra/meta.yaml

```diff
@@ -0,0 +1,25 @@
+{% set name = "pulchra" %}
+{% set version = "3.06" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+  number: 0
+  script: make && make install
```

Comment:
I guess ou need to copy the binary at least to $PREFIX/bin

Reply:
The install target in the Makefile includes that command